src/components/MessageBar/MessageBar.scss

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+//
+// Office UI Fabric
+// --------------------------------------------------
+// MessageBar Styles
+
+/*
+  Base template file
+*/
+
+$MessageBar-padding: 8px;
+
+.ms-MessageBar {
+  padding: $MessageBar-padding;
+  display: table;
+
+  .ms-MessageBar-icon,
+  .ms-MessageBar-text {
+    display: table-cell;
+    vertical-align: top;
+  }
+
+  .ms-MessageBar-icon {
+    padding-right: $MessageBar-padding;
+    font-size: 16px;
+  }
+
+  .ms-MessageBar-text {
+    @include ms-font-s;
+  }
+
+}
+
+.ms-MessageBar--info {
+  @include ms-bgColor-info;
+
+  .ms-MessageBar-icon {
+    @include ms-fontColor-neutralSecondaryAlt;
+  }
+
+  .ms-Icon:before {
+    content: '\e00a';  // ms-Icon--infoCircle
```

Comment:
Normally, that's what I'd do, but the nice thing about the way I built it is that we can switch the MessageBar type based on the main class.  Change ms-MessageBar--info to ms-MessageBar--error and you get the icon, color, and background switching automatically.  

I can go either way, but what do you think?